app/src/main/java/cm/aptoide/pt/view/BaseFragment.java

```diff
@@ -27,9 +27,9 @@
     }
   }
 
-  @Override public void onDestroyView() {
+  @Override public void onDestroy() {
     fragmentComponent = null;
-    super.onDestroyView();
+    super.onDestroy();
```

Comment:
I wouldn't advise to do this, as this can have a lot of implications. If you need to do this then maybe we should consider change the scope of the AppViewManager. But changing the scope of the appviewManager would mean changing the cache logic.
If the problem is just on rotation, you can use the savedInstance to control that.